### PR TITLE
B-1 An error occurs when register an user

### PR DESCRIPTION
**Pull request by xDAndrew**
Fixed registration page

### CODE REVIEW

File: PortfolioKeeper/Views/Account/Register.cshtml [hunk 1]
```diff
@@ -23,9 +23,10 @@
                             </div>
                             <div class="row">
                                 <div class="col-12 col-sm-6">
-                                    @Html.LabelFor(m => m.Email, new { @class = "col" })
-                                    @Html.TextBoxFor(m => m.Email, new { @class = "form-control" })
-                                    @Html.ValidationMessageFor(m => m.Email, "", new { @class = "text-danger col" })
+                                    @Html.Hidden("Id", Guid.NewGuid().ToString())
```
Comment (DashaDrahun) [inline PortfolioKeeper/Views/Account/Register.cshtml]: Maybe it's better to do this logic in AccountService (fix mapper and call userManager.CreateAsync() that do this work)?

Comment (xDAndrew) [reply PortfolioKeeper/Views/Account/Register.cshtml]: Done

File: PortfolioKeeper/Controllers/AccountController.cs [hunk 2]
```diff
@@ -79,6 +79,8 @@ public ActionResult Register()
             }
 
             var userDto = mapper.Map<RegisterViewModel, UserDto>(model);
+            userDto.Id = Guid.NewGuid().ToString();
```
Comment (stas-litvinenka) [inline PortfolioKeeper/Controllers/AccountController.cs]: you can put this inside automapper. no need to expand Controller action. Use .BeforeMap for example

**Comment by DashaDrahun** [comment]
Looks ok

**Pull request merged**